src/api/v1/paths/doughs/{doughId}.js

```diff
@@ -19,10 +20,24 @@ const get = async (req, res) => {
 };
 
 /**
- * stub of patch function
+ * Update a dough with the ID specified in the request body
  *
- * @returns null
+ * @type {RequestHandler}
  */
-const patch = async () => null;
+const patch = async (req, res) => {
+  try {
+    if (req.params.doughId !== req.body.data.id) {
```

Comment:
jsonapi [requires](https://jsonapi.org/format/#crud-updating-responses-409) this return and I didn't see a way to check this using the openapi spec